integration/env.go

```diff
@@ -125,24 +112,24 @@ func NewEnv(t *testing.T) *Env {
 		MaxInterval: ptypes.DurationProto(5 * time.Second),
 	})
 	if err != nil {
-		t.Fatalf("CreateDomain(): %v", err)
+		return nil, fmt.Errorf("env: CreateDomain(): %v", err)
 	}
 
 	mapID := domainPB.Map.TreeId
 	logID := domainPB.Log.TreeId
 	mapPubKey, err := der.UnmarshalPublicKey(domainPB.Map.GetPublicKey().GetDer())
 	if err != nil {
-		t.Fatalf("Failed to load signing keypair: %v", err)
+		return nil, fmt.Errorf("env: Failed to load signing keypair: %v", err)
 	}
 	vrfPub, err := p256.NewVRFVerifierFromRawKey(domainPB.Vrf.GetDer())
 	if err != nil {
-		t.Fatalf("Failed to load vrf pubkey: %v", err)
+		return nil, fmt.Errorf("env: Failed to load vrf pubkey: %v", err)
 	}
 
 	// Common data structures.
-	mutations, err := mutationstorage.New(sqldb)
+	mutations, err := mutationstorage.New(db)
 	if err != nil {
-		log.Fatalf("Failed to create mutations object: %v", err)
+		log.Fatalf("env: Failed to create mutations object: %v", err)
```

Comment:
Could this one be Errorf() like the previous ones you changed?

Reply:
yep, missed that one.